models/tt_transformers/tt/model_config.py

```diff
@@ -705,6 +711,15 @@ def __init__(
                 packer_l1_acc=False,
             )
 
+            # Configure data precision and math fidelity for tensors and kernels
+            self.model_config["COMPUTE_KERNEL_CONFIG_HIFI2"] = self.compute_kernel_config_hifi2
```

Comment:
Should the key be prepended with `MIXTRAL` like the next two in the code?
```suggestion
            self.model_config["MIXTRAL_COMPUTE_KERNEL_CONFIG_HIFI2"] = self.compute_kernel_config_hifi2
```